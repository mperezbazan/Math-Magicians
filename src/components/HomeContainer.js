import React from 'react';

const HomeContainer = () => {
  const title = 'Welcome to our page!';
  const firstParagraph = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent et nisi vitae augue sagittis rutrum nec at est. In augue mi, venenatis eget mi ultrices, varius iaculis enim. Ut posuere, leo in congue ullamcorper, nulla lacus commodo ligula, et faucibus mauris sem quis sapien. Etiam hendrerit justo mauris, quis egestas mi maximus vel. Aliquam in purus convallis, commodo est at, ullamcorper elit. Proin consectetur, dolor in tristique placerat, est quam posuere urna, dapibus placerat nulla ex eu urna. Donec eu fringilla odio. Donec consequat ut dui at mattis. Sed bibendum lobortis orci et efficitur. Maecenas in neque molestie nisl pellentesque sodales. Donec fringilla eget magna et cursus. Suspendisse eu lorem est. Vestibulum scelerisque varius vulputate. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.l';
  const secondParagraph = 'Sed non ipsum sapien. Pellentesque blandit sodales justo pretium tristique. Curabitur sagittis diam velit, at auctor augue euismod ut. Duis efficitur eu neque sit amet iaculis. Nulla sagittis auctor imperdiet. Vestibulum sit amet lorem imperdiet, consequat nisl in, vehicula ante. Morbi eget luctus est. Aenean facilisis nunc vel sem volutpat maximus. Morbi vel maximus nunc. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Mauris ut ullamcorper nisi, consectetur mattis urna. Quisque arcu leo, condimentum non tincidunt et, sollicitudin et augue.';
  return (
    <div className="home-container">
      <h1 className="page-title">{title}</h1>
      <p className="home-text">{firstParagraph}</p>
      <p className="home-text">{secondParagraph}</p>
    </div>
  );
};

export default HomeContainer;
