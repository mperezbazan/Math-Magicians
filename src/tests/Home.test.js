import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import HomeContainer from '../components/HomeContainer';

test('Render Home Container ', () => {
  render(<HomeContainer />);
  const heading = screen.getByRole('heading');
  expect(heading).toBeInTheDocument();
  expect(heading).toHaveTextContent('Welcome to our page!');
});

test('Match Home Snapshot', () => {
  const tree = renderer.create(<HomeContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});
