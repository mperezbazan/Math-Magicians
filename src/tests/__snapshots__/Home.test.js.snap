// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match Home Snapshot 1`] = `
<div
  className="home-container"
>
  <h1
    className="page-title"
  >
    Welcome to our page!
  </h1>
  <p
    className="home-text"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent et nisi vitae augue sagittis rutrum nec at est. In augue mi, venenatis eget mi ultrices, varius iaculis enim. Ut posuere, leo in congue ullamcorper, nulla lacus commodo ligula, et faucibus mauris sem quis sapien. Etiam hendrerit justo mauris, quis egestas mi maximus vel. Aliquam in purus convallis, commodo est at, ullamcorper elit. Proin consectetur, dolor in tristique placerat, est quam posuere urna, dapibus placerat nulla ex eu urna. Donec eu fringilla odio. Donec consequat ut dui at mattis. Sed bibendum lobortis orci et efficitur. Maecenas in neque molestie nisl pellentesque sodales. Donec fringilla eget magna et cursus. Suspendisse eu lorem est. Vestibulum scelerisque varius vulputate. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.l
  </p>
  <p
    className="home-text"
  >
    Sed non ipsum sapien. Pellentesque blandit sodales justo pretium tristique. Curabitur sagittis diam velit, at auctor augue euismod ut. Duis efficitur eu neque sit amet iaculis. Nulla sagittis auctor imperdiet. Vestibulum sit amet lorem imperdiet, consequat nisl in, vehicula ante. Morbi eget luctus est. Aenean facilisis nunc vel sem volutpat maximus. Morbi vel maximus nunc. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Mauris ut ullamcorper nisi, consectetur mattis urna. Quisque arcu leo, condimentum non tincidunt et, sollicitudin et augue.
  </p>
</div>
`;
